src/inverseRobot.js

```diff
@@ -4,10 +4,23 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  const keysRobot = Object.values(robot);
```

Comment:
The name 'keysRobot' is misleading, it should be 'valuesRobot' as it stores object values.

Reply:
I understand why you named it `keys` - it will be the keys in the future - but you might show it in the name as well. Anyway not critical here - approved